<launch>
<env name="ROSCONSOLE_FORMAT" value="[${severity}] [${node}]: ${message}" />
    <env name="ROSCONSOLE_CONFIG_FILE" value="$(find robot_main)/custom_rosconsole.config" />

    <include file="$(find rosbot_ekf)/launch/all.launch">
        <arg name="rosbot_pro" value="true" />
    </include>

    <include file="$(find rplidar_ros)/launch/rplidar_a3.launch" />

    <node pkg="tf" type="static_transform_publisher" name="laser_broadcaster" args="0 0 0.2 3.14 0 0 base_link laser 100" />

    <node pkg="robot_main" type="robot_main_node" name="robot_main_node" output="screen" required="true"/>
    <node pkg="robot_driver" type="get_correction_server.py" name="get_correction_server" output="screen" required="true" />
    <node pkg="robot_gui" type="start" name="robot_gui" output="screen" required="true"/>
</launch>
